### PR TITLE
Unify settings names 

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

File: bears/js/JSHintBear.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def generate_config(filename, file,
         :param environment_phantom:
             This option defines globals available when your core is running
             inside of the PhantomJS runtime environment.
-        :param maxerr:
+        :param max_errors:
```
Comment (sils1297) [inline bears/js/JSHintBear.py]: can we rather not expose this and pass something relatively close to infinity instead? :P

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/624/commits/f602a720d9f7b6401933b62eb7bd1d9a6e548255

File: bears/coffee_script/CoffeeLintBear.py [hunk 2]
```diff
@@ -209,7 +209,7 @@ class Foo
             ``windows``.
         :param no_this:
             Prohibits ``this``. Use ``@`` instead.
-        :param no_decr_or_incrementation_operators:
+        :param allow_increment:
             Prohibits the use of increment and decrement arithmetic operators.
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: wrong description

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/8b72a1025f5538ea2665e694531739a876b005d8

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/624/commits/937d19a0854c54b175f5375e36b1c792f74f7ffe

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/68dcaa2b10f06a26e28e6f625f11fc1e363d7fbf

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/e89321a364c8645fda743bb6287ceac05fdf1a9d

File: bears/coffee_script/CoffeeLintBear.py [hunk 3]
```diff
@@ -207,7 +207,7 @@ class Foo
         :param consistent_line_endings_style:
             The option to ``line_endings``, its value is either ``unix`` or
             ``windows``.
-        :param no_this:
+        :param allow_this_statements:
             Prohibits ``this``. Use ``@`` instead.
```
Comment (sils1297) [inline bears/coffee_script/CoffeeLintBear.py]: please update your doc comments!

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/a3681aa1071525d94823176249315a461473203b

**Comment by sils1297** [comment]
Hey you didn't read my comments! You sometimes invert the meaning of parameters but you don't edit the description!

File: bears/js/JSHintBear.py [hunk 4]
```diff
@@ -262,7 +262,7 @@ def generate_config(filename, file,
         :param maxdepth:
             This option lets you control how nested do you want your blocks to
             be.
-        :param maxparams:
+        :param max_formal_parameters:
             Maximum number of formal parameters allowed per function.
```
Comment (sils1297) [inline bears/js/JSHintBear.py]: what is a formal parameter?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/2ae2bbdd8e183b82bcf1517024c2d925b505e1df

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/f28bee8d132a87aff48a38cf0910005e9c101b1c

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/e63bffe06bd9eb52bad1892a6a946bf74adc3ca3

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/e68da2a9ec07b71f44de7db3e2b154ec53c451b4

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/de46e5d489314cafe944ed3006d6c35e1fc75507

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/1f5d96e9222716ffa39ef146b618cb66169ccce1#diff-a321f9a8f5f440c140740f55883d07e7L116

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/af121ad69bfdd2651031a5323ced10c0ee9a07b2

**Comment by Asnelchristian** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/624/commits/fe05811f9d9b8e6c96145b39f6d599fe2689e4e9 https://github.com/coala-analyzer/coala-bears/pull/624/commits/22c5cbd47d3d625f2a8c4eceedcb34e92b0a06b4 https://github.com/coala-analyzer/coala-bears/pull/624/commits/b21e01d81efe278382bc1e1821dad1b49bce78e8 https://github.com/coala-analyzer/coala-bears/pull/624/commits/94d9584b49d95d43f858f308ca9861db13566b45 https://github.com/coala-analyzer/coala-bears/pull/624/commits/2a322c9f3bf95470e4969e6e3e93b555fec3aef8 https://github.com/coala-analyzer/coala-bears/pull/624/commits/8b28526f22a20a07eb9c49b48a8accfa4ebdeff4 https://github.com/coala-analyzer/coala-bears/pull/624/commits/4216bcf94cc4ecf09e2c14d168a3c179b9a912f8

File: bears/js/JSHintBear.py [hunk 5]
```diff
@@ -100,7 +100,7 @@ def generate_config(filename, file,
                         maxerr: int=50,
                         maxstatements: bool_or_int=False,
                         maxdepth: bool_or_int=False,
-                        maxparams: bool_or_int=False,
+                        max_parameters: bool_or_int=False,
```
Comment (sils1297) [inline bears/js/JSHintBear.py]: I'd set it to 8 or so by default?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/624/commits/7a2b9cbb518d648a069bd31788d4637ee7aacd23

**Comment by sils1297** [comment]
rebase needed btw.?

**Comment by Asnelchristian** [comment]
reack  https://github.com/coala-analyzer/coala-bears/pull/624/commits/9ddc2e85e22de6282360c429e37caf28d2d8a8f4 https://github.com/coala-analyzer/coala-bears/pull/624/commits/678a7faa32df2cba9a7b4186c961f416f0ba897f https://github.com/coala-analyzer/coala-bears/pull/624/commits/c3fea8a002cfc69421062b5a5c72f3e8ba5cda65 https://github.com/coala-analyzer/coala-bears/pull/624/commits/a50328a8906c69f79de3d0e065434faee1f1e3f9 https://github.com/coala-analyzer/coala-bears/pull/624/commits/6bfb79ad0ced74cf632e00a5a8d48ad1526094f4 https://github.com/coala-analyzer/coala-bears/pull/624/commits/58e365ce291833dc8678a17d60c372a7795e3fab https://github.com/coala-analyzer/coala-bears/pull/624/commits/59099a95dbe395793ecbb78f290e6add2eb497e9  https://github.com/coala-analyzer/coala-bears/pull/624/commits/e3fca3bb74bffdaf8982031849c51b9ff0134790  https://github.com/coala-analyzer/coala-bears/pull/624/commits/fb56cc449e12a669966178a24c43d382cd3a27b2

**Comment by Asnelchristian** [comment]
@rultor merge

File: bears/js/JSHintBear.py (hunk 5 above)
Comment (Asnelchristian) [inline bears/js/JSHintBear.py]: i think you should leave the default value provided by the linter here

**Pull request merged**